jujugui/static/gui/src/app/components/account/credentials/credentials.js

```diff
@@ -56,6 +58,9 @@ class AccountCredentials extends React.Component {
           console.error(message, error);
           return;
         }
+        // Add the cloud name into the clouds object so it can be used
+        // by other components.
+        Object.keys(clouds).forEach(name => clouds[name].name = name);
```

Comment:
Perhaps we should do this directly at the api level? [shrug]

Reply:
yeah that's a good idea, I'll make a note to do a follow-up and update elsewhere in the app that this is being done.